WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -101,15 +100,16 @@ public WalletViewModel(UiContext uiContext, IWalletModel walletModel, Wallet wal
 		this.WhenAnyValue(x => x.Settings.PreferPsbtWorkflow)
 			.Do(x => this.RaisePropertyChanged(nameof(PreferPsbtWorkflow)))
 			.Subscribe();
+
+		this.WhenAnyValue(x => x.WalletModel.Name).BindTo(this, x => x.Title);
+		this.WhenAnyValue(x => x.WalletModel.Name).BindTo(this, x => x.WalletName);
```

Comment:
Why do we need a title and wallet name too?

Reply:
Yes, that's redundant. I've updated the code to just keep Title (needed by `RoutableViewModel`)